source/collaborate/collaborate-within-connected-microsoft-teams.rst

```diff
@@ -7,34 +7,61 @@ Collaborate within Microsoft Teams
 
 1. Log into Mattermost using your credentials. 
 2. When you log in, you’ll be prompted to enter your Microsoft Teams user information, including your Microsoft Teams email address and your Microsoft Teams password.
 
-Mattermost will tell you when you’ve successfully connected your account. 
+.. note::
+
+  - To change which accounts are linked, in Mattermost, disconnect the current account by running the slash command ``/msteams disconnect`` within the Mattermost message text box. Then, re-connect a new Mattermost account to a new Microsoft Teams account by running the Mattermost slash command ``/msteams connect``.
+  - If your Microsoft Teams user account is removed from Microsoft Teams, your linked Mattermost user account will also be removed the next time user accounts are synchronized across the two platforms.
+
+Mattermost will confirm when your account is connected, and prompt you to select your primary platform.
+
+Select your preferred platform
+-------------------------------
+
+Once you've connected your Mattermost account to your Microsoft Teams account, Mattermost prompts you to choose which platform you'll use more often: `Mattermost <#mattermost-is-your-preferred-platform>`_ or `Microsoft Teams <#microsoft-teams-is-your-preferred-platform>`_. Mattermost optimizes your experience based on the platform you prefer to use.
 
 .. note::
 
-  - If you're not prompted to connect your accounts, display the prompt in a browser window by running the Mattermost slash command ``/msteams connect`` in Mattermost. Follow the link to connect your accounts.
-  - To change which accounts are linked, in Mattermost, run the slash command ``/msteams disconnect``, then connect your Mattermost account to a new Microsoft Teams account.
-  - If a Microsoft Teams user account is removed from Microsoft Teams, the linked Mattermost user account is also removed the next time user accounts are synchronized.
+  - If you're not prompted to connect your accounts, run the Mattermost slash command ``/msteams connect`` within the Mattermost message text box to display the prompt in a browser window. Follow the link to connect your accounts.
+  - If you select **Skip for now**, Mattermost is selected as your primary platform. You can change this preference any time by selecting your profile picture and going to **Settings > MS Teams Settings**.
+
+Mattermost is your preferred platform
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When Mattermost is preferred, you'll receive Microsoft Teams messages and notifications in Mattermost.
+
+We strongly recommend that you disable notifications in Microsoft Teams. If you don't, you'll receive duplicate message notifications in both platforms. See the `Microsoft Teams notifications documentation <https://support.microsoft.com/en-us/office/manage-notifications-in-microsoft-teams-1cc31834-5fe5-412b-8edb-43fecc78413d>`__ for details.
```

Comment:
FYI with **Select sync** this "strongly suggest" may be deprioritized as the issue of dual notification is significantly reduced